app/src/Screens/RegisterProject.svelte

```diff
@@ -123,7 +123,7 @@
       {#if step === 4}
         {#if response}
           <TransactionSummaryStep
-            name={response.data.registerProject.messages[0].name}
+            name={response.data.registerProject.messages[0].project_name}
```

Comment:
You'd have to change the name of the prop like you did it in the component for this to work. I'll fix it in a followup. Will also write tests for this once the whole Org registration feature is a bit more than a POC.

Reply:
Going to address this here: https://github.com/radicle-dev/radicle-upstream/pull/161.